docs/usage/service-worker.md

```diff
@@ -0,0 +1,417 @@
+(using_from_service_worker)=
+
+# Using Pyodide in a service worker
+
+This document describes how to use Pyodide to execute Python scripts in a service worker. Compared to typical web workers, service workers are more related to acting as a network proxy, handling background tasks, and things like caching and offline. See [this article](https://web.dev/workers-overview/#use-cases) for more info.
```

Comment:
Can you hard wrap this to 80ish characters? Some lines can be a bit longer but we don't want 100+ character long lines.

Reply:
done!